### PR TITLE
🐛 [Story performance] Apply last animation frames on prerender

### PR DESCRIPTION
**Pull request by mszylkowski**
On prerender stories, the LCP is being delayed on fade-in animations by the last animation frames only being applied after the page becomes visible. We want to apply the animation frames in prerender mode (in the buildCallback).

Also, the `renderOpenAttachmentUI` was being called twice, which is unnecessary (and removed test on `beforeVisible` since it's not used anymore).
image

**TODO: Add test for importing the page attachment extension on auto-ads**

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-page.js
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-page.js:545]: cc @calebcordry to help testing this for Ads

**Review by gmajoulet — APPROVED** [review]
LGTM for me but I believe you're breaking Ads, please wait for @calebcordry to also approve before merging

File: extensions/amp-story/1.0/amp-story-page.js
Comment (mszylkowski) [reply extensions/amp-story/1.0/amp-story-page.js:545]: I tried following the logic and doesn't seem it would change anything: `beforeVisible` is called at the end of layouotCallback inside a `Promise.all` but immediately before that there's a `renderOpenAttachmentUI` so the two calls to `renderOpenAttachmentUI` happen synchronously with each other.

The other call to `beforeVisible` happens in layoutStory with just the first initial page of the story, which I don't think can eever be an ad.

Will still wait for @calebcordry to confirm this.

Comment (calebcordry) [reply extensions/amp-story/1.0/amp-story-page.js:545]: Good catch! Matias and I just manually tested and confirmed this does break the new CTA rendering. I will look to see if I can have ads install the `page-outlink` extension instead of doing it in `beforeVisible`. If this works as expected it should be safe to delete here.

**Comment by mszylkowski** [comment]
Converting to draft until ads load the page-attachment properly, then will open again

File: extensions/amp-story-auto-ads/0.1/story-ad-ui.js
Comment (calebcordry) [inline extensions/amp-story-auto-ads/0.1/story-ad-ui.js:219]: Services is not defined :)

Comment (mszylkowski) [reply extensions/amp-story-auto-ads/0.1/story-ad-ui.js:219]: Forgot an "import Services" somewhere, will add it

**Review by calebcordry — APPROVED** [review]

**Pull request merged**
